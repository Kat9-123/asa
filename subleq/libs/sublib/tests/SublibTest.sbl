!J .main
string2 -> &"is cool! " 0


#sublib

string -> &"Subleq " 0

i -> 1

.main ->
    !IO::Print string
    !IO::Print string2
    !IO::Print string
    !IO::Print string2
    i -= ZERO 0xFFFF
    i -= ONE
    !J .main
    !Halt