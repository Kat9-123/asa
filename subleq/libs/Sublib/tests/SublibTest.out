Subleq is cool! Subleq is cool! Subleq is cool! Subleq is cool! 